src/main.rs

```diff
@@ -69,6 +75,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         println!("Gossip Stopped");
     });
 
+    // TODO: config to enable/disable running this fetcher
+    // TODO: add start_from as a parameter
+    let mut fetcher = Fetcher::new(685400u64);
```

Comment:
Another option is to just fetch the first 100 or so transfers or only do it for our fids. Main benefit is the data is going to be bounded and consistent across runs.

Reply:
Interesting, perhaps both `from` and `to` parameters? I guess these will be temporary.

If we place an upper bound though, we won't be exercising the codepath that relaxes into the 5s polling delay that discovers new transfers. Although perhaps that's better handled by test cases (which we also don't have).

To get consistency between runs we might need a persistence layer as well, as a slow network may not fetch the same set of IDs before we start asking for them.

I'm tempted to defer the decision until we have a config system and things are further along.